prowler/providers/aws/services/iam/iam_user_console_access_unused/iam_user_console_access_unused.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.iam.iam_client import iam_client
+
+maximum_expiration_days = iam_client.audit_config.get("max_console_access_days")
```

Comment:
Putting this line there makes the tests to fail, you have to move it under line 10.

And set a default value with the same value in `config.yaml`. 

https://docs.prowler.cloud/en/latest/developer-guide/checks/#using-the-audit-config